app/views/content_items/document_collection.html.erb

```diff
@@ -42,6 +46,9 @@
       </ol>
     <% end %>
   </div>
+  <div data-sticky-element class="govuk-sticky-element">
```

Comment:
Use of the class name `govuk-sticky-element` hints that it's a component, which it isn't.